StreetSmartArcMap/Logic/StreetSmartApiWrapper.cs

```diff
@@ -930,16 +930,6 @@ private void StreetSmartAPI_ViewerRemoved(object sender, IEventArgs<IViewer> e)
 
         public void SetOverlayDrawDistance(int distance, esriUnits mapUnits)
         {
-            switch (mapUnits)
-            {
-                // For now skip the conversion calculation, because it is not needed if Street Smart works in a feet coordinate system
-//                case esriUnits.esriFeet:
-//                    distance = (int)Math.Round(distance * 3.280839895, 0);
-//                    break;
-
-                default: break;
-            }
-
```

Comment:
The problem you had with the merge conflicts that occurred was because after you tried and merged the last pull request, you no longer did any fetch / pull.

I have already solved the merge conflicts for you

That solved, among other things, the problem you had with the absence of the help buttons file.

In the latest version I already commented on this piece of code because it was not necessary. (with a comment line above it with the reason)

The: **ArcMap.Document.FocusMap.MapUnits** units that go into this function relate to the visibility units of ArcMap, so that ArcMap can show in feet while working with meter coordinate systems, or vice versa.

The question is whether we should now also incorporate this into the ArcMap integration. The latest version of the API also supports this functionality now.

The GlobeSpotter ArcMap integration also supports this functionality.